.github/workflows/copilot_deploy.yml

```diff
@@ -106,15 +118,15 @@ jobs:
 
     - name: Inject replacement image into manifest
       run: |
-        yq -i '.image.location = "ghcr.io/communitiesuk/funding-service-design-notification:${{ github.ref_name == 'main' && 'latest' || github.ref_name }}"'  copilot/fsd-notification/manifest.yml
+        yq -i '.image.location = "ghcr.io/communitiesuk/funding-service-design-notification:${{ github.ref_name == 'main' && 'latest' || needs.tag_version.outputs.version_to_tag }}"'  copilot/fsd-notification/manifest.yml
 
     - name: Copilot ${{ matrix.value }} deploy
       id: deploy_build
       run: |
         copilot svc deploy --env ${{ matrix.value }} --app pre-award
 
   post_deploy_tests:
-    if: inputs.environment == 'test' || inputs.environment == ''
+    if: inputs.environment == 'test' || inputs.environment == 'dev' || inputs.environment == ''
```

Comment:
If we deploy to production is this going to run post-deploy tests there?  Is that what we want?

Reply:
Hmm, cos of line 139, I guess it would do. No, we don't.